xtask/src/docgen.rs

```diff
@@ -95,14 +94,14 @@ pub fn lang_features() -> Result<String, DynError> {
                 .to_owned(),
             );
         }
-        row.push(
-            lc.language_servers
-                .iter()
-                .filter_map(|ls| config.language_server.get(&ls.name))
-                .map(|s| md_mono(&s.command.clone()))
-                .collect::<Vec<_>>()
-                .join(", "),
-        );
+        let mut v: Vec<String> = lc.language_servers
+                            .iter()
+                            .filter_map(|ls| config.language_server.get(&ls.name))
+                            .map(|s| md_mono(&s.command.clone()))
+                            .collect::<Vec<_>>();
```

Comment:
```suggestion
                            .collect::<HashSet<_>>();
```
Is what I meant

Reply:
Hmm, actually substituting the `join` functionality is a pain until `intersperse lands`

Reply:
Yep, had to convert to slice first to join them as a string.. Does Rust have a stringify method to use? might be simpler way to convert list to comma string.

Reply:
If your happy with it like this can we merge it?